src/components/EventCard.tsx

```diff
@@ -0,0 +1,61 @@
+import styles from "../css/EventCard.module.css";
+
+const EventCard = ({ title, startDateTime, location }) => {
+	return (
+		<div className={`${styles["event-card"]}`}>
+			<div className={`${styles["left-container"]}`}>
+				<div className={`${styles["date-container"]}`}>
+					<h3>Date</h3>
```

Comment:
Be careful with the section heading tags. If you look at the structure you currently have, it goes from `h3` to `h2` in the code. While it visually may not look like that, when viewing the site with screen readers, or when it gets crawled by Google for SEO, the tags will be out of order. To remediate this, I would first add an `h1` tag at the top of the page, and then leave the event title as `h2` tags. The date, if it eventually should house a date, should use the `time` HTML element instead.

Reply:
`time` tag documentation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
@quinnmcphail Thank you for this comment regarding SEO best practices. I'm learning how super important they are. To fix this, I added an `h1` tag at the top of the EventsPage component and committed that change to this pull request.

Reply:
I changed both of the time-related elements to <time> tags. I don't know how the group wants to handle transforming the ISO dates from the markdown files.  That's why I simply put placeholder text there, but eventually we'll convert the ISO dates to readable text (with a JS function?) and make the date elements dynamic. I added the "dateTime" attribute to the <time> tags, although the value for the attribute will eventually be dynamic too.